transformation/transformationPlugins/jsonMessageTransformers/jsonJinjavaTransformer/src/main/resources/jinjava/common/featureEnabled.j2

```diff
@@ -0,0 +1,27 @@
+{%- macro is_enabled(features, path) -%}
+    {%- if features == None -%}
+      true
+    {%- else -%}
+        {%- set ns = namespace(value=features) -%}
+        {%- set debug = namespace(log=[]) -%}
```

Comment:
How does this debug log get used/printed?

Reply:
In this case, we'll accumulate stuff in that list and then do nothing with it.  If we choose to, we can emit its value (or derivatives) in the output - which would have an impact on how the result of is_enabled would be interpreted.  It's pretty much a one-time use solution - once you emit something and muddy the signal, the rest of the templating will be off.

Another option is to pass arguments into the dynamic function, which gets them back into a java context.  From there, we could debug, etc.  That might be something very much worth adding (just thought of it).

Reply:
I now have functions to log values through the Slf4j configurations for the jinjava transformer package.